src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/config/InterceptorConfig.java

```diff
@@ -4,16 +4,23 @@
 import org.springframework.lang.NonNull;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import uk.gov.companieshouse.api.interceptor.TokenPermissionsInterceptor;
+import uk.gov.companieshouse.limitedpartnershipsapi.interceptor.CustomUserAuthenticationInterceptor;
 import uk.gov.companieshouse.limitedpartnershipsapi.interceptor.LoggingInterceptor;
 
-
 @Configuration
 public class InterceptorConfig implements WebMvcConfigurer {
 
+    private static final String PARTNERSHIP = "/transactions/**/partnership**";
```

Comment:
Do we need the "**" on the end of this pattern string or even the "/partnership**"?

In ROE we had just:

   `static final String TRANSACTIONS = "/transactions/**";`

But maybe that was for something different?

Reply:
This was suggested by @markpit, Mark what do you think?

Reply:
It looks like it covers all extensions of .../partnership

Reply:
In ROE i think we applied the user auth to all the transactions, but it checks if it uses an api key and will bypass if it does. So we could follow the same pattern. So maybe follow the same approach, otherwise we will have to write separate Create / read etc interceptors unless there is a CRUD one we can re-use.